content/en/docs/reference/kubectl/_index.md

```diff
@@ -136,7 +136,7 @@ Operation       | Syntax    |       Description
 `diff`        | `kubectl diff -f FILENAME [flags]`| Diff file or stdin against live configuration.
 `drain`    | `kubectl drain NODE [options]` | Drain node in preparation for maintenance.
 `edit`        | <code>kubectl edit (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) [flags]</code> | Edit and update the definition of one or more resources on the server by using the default editor.
-`events`      | `kubectl events` | List events
+`events`      | `kubectl get events` | List events
```

Comment:
You know the main website is for 1.26 which supports `kubectl events`, right?

Reply:
Thanks for indicating. I am able to get version specific docs now. I will cancel the pull request.